src/TxHistory/AssetList/AssetItem/AssetItem.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import {action} from '@storybook/addon-actions'
+import {storiesOf} from '@storybook/react-native'
+import React from 'react'
+import {Text, View} from 'react-native'
+
+import {mocks, QueryProvider} from '../../../../storybook'
+import {Spacer} from '../../../components'
+import {SelectedWalletProvider} from '../../../SelectedWallet'
+import {primaryTokenInfo} from '../../../yoroi-wallets'
+import {AssetItem} from './AssetItem'
+
+storiesOf('AssetItem', module).add('Gallery', () => (
+  <QueryProvider>
+    <SelectedWalletProvider wallet={mocks.wallet}>
+      <View style={{flex: 1, justifyContent: 'center', padding: 16}}>
+        <Text>Fungible primary token</Text>
+
+        <AssetItem tokenInfo={primaryTokenInfo.mainnet} onPress={action('onPress')} />
```

Comment:
mocks.wallet.testnet.primaryTokenInfo

Reply:
or maybe we should get rid of testnet mocks altogether

Reply:
I'd say move mocks data as mainnet